### DISCUSSION TITLE
Fix my project error

### DISCUSSION THREAD

**Discussion by cuzmuc** [cloudbet/docs#91] [Q&A]
Hello, everyone!
I am Cuzmuc Ghennadii from Moldva. I hope you'r doing well.
I have a trouble with placing a bet programmatically on Cloudbet using his API.
I nearly finished my project at last using node.js, but I have the only important error.

These are input and output of my project:
- Input
Bet Info: {
  acceptPriceChange: 'BETTER',
  currency: 'BTC',
  eventId: '407079709',
  marketUrl: 'tennis.set_handicap/home?handicap=1.5',
  side: 'BACK',
  price: '1.67',
  stake: '0.0002',
  referenceId: '04975f65-3430-4a3f-8d58-0259a2abc962'
}

- Output
400 Bad Request: Invalid request format or parameters. {
  referenceId: '04975f65-3430-4a3f-8d58-0259a2abc962',
  price: '1.67',
  eventId: '407079709',
  marketUrl: 'tennis.set_handicap/home?handicap=1.5',
  side: 'BACK',
  currency: 'BTC',
  stake: '0.0002',
  createTime: '',
  status: 'MARKET_SUSPENDED',
  returnAmount: '',
  eventName: '',
  sportsKey: '',
  competitionId: '0',
  categoryKey: '',
  customerReference: 'bdf7b3c5-9bbd-46e9-b436-99d49269e768',
  error: ''
}

I have no experience in this field, so I want you to help me a lot possible to fix quickly.
Thanks for concerning my suggestion!

Best regard,
Cuzmuc Ghennadii

**Comment by kgravenreuth** [comment]
Hello @cuzmuc ,

As per your provided details, the event ID is `407079709` which doesn't exist in our system. Hence, the request failed in our system. We would suggest you to carefully review all the parameters sent in your request to ensure that accurate information is being sent in your Betting requests via the Cloudbet Trading API. 

Please refer to our previous discussion in #90 where I had provided several resources to get you started with the Cloudbet API, thanks!

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by cuzmuc** [comment]
Hi there,

It's my pleasure to have your response to my question!
I tried a lot for about 20 days to know why, but I couldn't find out why.
I could know the reason exactly after receiving your email.
Thanks again.

In spite of your kind explanations, I still have something unclear to place
bets programmatically because I am not experienced in this field.
The input data I used for my betting was from the API of betburger.com(
https://rest-api-pr.betburger.com/api/v1/arbs/bot_pro_search).

I want to know why I used wrong information, and how I can fix this problem.
Of course, I know this may not be your field or role, but I want to get
help from you to solve this problem.
I am sure I will solve this problem perfectly under your help within a few
days.
Looking forward to hearing from you!

Best regards,
Cuzmuc Ghennadii.

**Comment by cuzmuc** [comment]
[image: image.png]

Thanks for your exact and kind support!

I wouldn't solve my problem without your help.

I  personally  appreciate the support team in  Cloudbet, Betburger for
supporting me.

I hope you have greater success in your business and health.

Thanks, again.
